src/components/Popups/ReplyField/ReplyField.scss

```diff
@@ -1,12 +1,27 @@
+@import '~box-ui-elements/es/styles/variables';
 @import '~draft-js/dist/Draft.css';
 
 .ba-ReplyField {
-    div[contentEditable='true'] {
+    .public-DraftEditorPlaceholder-hasFocus,
+    .public-DraftEditorPlaceholder-root {
+        padding: 12px;
+        color: $bdl-gray-62;
+    }
+
+    // Element selector needed to override inherited BUIE styles
+    div[contentEditable='true'],
+    div[contentEditable='false'] {
```

Comment:
Isn't the `&:disabled` covered the disabled state style?

Reply:
I don't see that attribute being set, no. I've pushed another commit with more fixes.